src/core/helpers.js

```diff
@@ -172,7 +172,8 @@ wb.addSkipLink = function( text, attr, isBtn, isLast ) {
 // Escapes the characters in a string for use in a jQuery selector
 // Based on https://totaldev.com/content/escaping-characters-get-valid-jquery-id
 wb.jqEscape = function( selector ) {
-	return selector.replace( /([;&,.+*~':"\\!^/#$%@[]()=>\|])/g, "\\$1" );
+	// eslint-disable-next-line no-useless-escape
+	return selector.replace( /([;&,\.\+\*\~':"\\\!\^\/#$%@\[\]\(\)=>\|])/g, "\\$1" );
```

Comment:
FWIW jqEscape()'s functionality is built-into [modern-day JavaScript via ``CSS.escape()``](https://developer.mozilla.org/en-US/docs/Web/API/CSS/escape) and [jQuery 3.x via ``$.escapeSelector()``](https://api.jquery.com/jQuery.escapeSelector/).

So... I was thinking of putting ``CSS.escape()`` into use here, but unfortunately IE11 doesn't support it and I didn't feel like it'd be wise to intentionally break it.